src/hotspot/share/memory/virtualspace.hpp

```diff
@@ -76,17 +76,11 @@ class ReservedSpace {
 
   // Splitting
   // This splits the space into two spaces, the first part of which will be returned.
-  // If split==true, the resulting two spaces can be released independently from each other.
-  //  This may cause the original space to loose its content.
-  //  They also will be tracked individually by NMT and can be tagged with different flags.
-  //  Note that this may cause the original space to loose its content.
-  // If split==false, the resulting space will be just a hotspot-internal representation
-  //  of a sub section of the underlying mapping.
-  ReservedSpace first_part(size_t partition_size, size_t alignment, bool split = false);
+  ReservedSpace first_part(size_t partition_size, size_t alignment);
   ReservedSpace last_part (size_t partition_size, size_t alignment);
 
   // These simply call the above using the default alignment.
-  inline ReservedSpace first_part(size_t partition_size, bool split = false);
+  inline ReservedSpace first_part(size_t partition_size/*, bool split = false*/);
```

Comment:
Forgot to remove the comment code `/*, bool split = false*/`?

Reply:
Yes, thanks for the review.